tests/ecmp/test_fgnhg.py

```diff
@@ -334,9 +678,11 @@ def test_fg_ecmp(common_setup_teardown, ptfadapter, ptfhost):
     duthost, cfg_facts, router_mac, net_ports = common_setup_teardown
 
     # IPv4 test
-    port_list, ip_to_port, bank_0_port, bank_1_port = setup_test_config(ptfadapter, duthost, ptfhost, cfg_facts, router_mac, net_ports, DEFAULT_VLAN_IPv4, PREFIX_IPv4)
-    fg_ecmp(ptfhost, duthost, router_mac, net_ports, port_list, ip_to_port, bank_0_port, bank_1_port, PREFIX_IPv4) 
+    port_list, ipv4_to_port, bank_0_port, bank_1_port = setup_test_config(ptfadapter, duthost, ptfhost, cfg_facts, router_mac, net_ports, DEFAULT_VLAN_IPv4)
```

Comment:
can this be module level fixture with param ? We can do clean up as part of fixture. Not able to see where cleanup is happening.

Reply:
common_setup_teardown in line 678 does cleanup after test is completed. This is a similar model to what is done in a few other tests as well, eg: test_bgp_speaker.